AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/SettingsContext.cs

```diff
@@ -25,9 +25,9 @@ public SettingsContext(DbContextOptions<SettingsContext> options)
     protected override void OnModelCreating(ModelBuilder modelBuilder)
     {
         // Declare keys and relationships here
-        modelBuilder.Entity<User>().HasKey(u => u.UserId);
+        modelBuilder.Entity<User>().HasKey(u => u.Id);
 
-        modelBuilder.Entity<SettingGroup>().HasKey(s => s.SettingGroupId);
+        modelBuilder.Entity<SettingGroup>().HasKey(s => s.Id);
```

Comment:
We might want to do another migration to add a couple more indexes to these but should be fine for now. We're gonna have realistically very little data in the database